bigchaindb/common/utils.py

```diff
@@ -46,3 +49,27 @@ def deserialize(data):
             string.
     """
     return rapidjson.loads(data)
+
+
+def validate_txn_obj(obj_name, obj, key, validation_fun):
+    backend = bigchaindb.config['database']['backend']
+
+    if backend == 'mongodb':
+        data = obj.get(key, {}) or {}
```

Comment:
Can you explain why the `or {}` is needed? Won't `obj.get(key, {})` return `{}` if `obj[key]` doesn't exist? I don't know why `obj.get(key, {})` would ever be falsy.

Reply:
@ttmc  In the `GENESIS` block the value of 'data' is None i.e. we `{'data':None}` so `obj.get(key,{})` resolves to `None`. So, to avoid that I added `or {}`